state/state.go

```diff
@@ -24,13 +24,19 @@ import (
 // HelmState structure for the helmfile
 type HelmState struct {
 	BaseChartPath      string
+	Helm               HelmSpec         `yaml:"helm"`
```

Comment:
I'm still thinking about what would be the best key name here. Assuming it is possible to override what are specified under the `helm:` setting, how about naming it `helmDefaults:` so that it would be clearer that you can override?

Reply:
helmDefaults sounds good to me. Will make changes